### PR TITLE
Created file variables.md.

### PR DESCRIPTION
**Pull request by saboorElahi**
Added a simple C++ program which uses integer variable.

### CODE REVIEW

File: src/pages/cplusplus/variable.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Lets discuss something know as variables. Variables are like a bucket. You can put something in it and then change it
```
Comment (erictleung) [inline src/pages/cplusplus/variable.md]: Can you add the following YAML header above this line? Thanks!

```yaml
---
title: Variables
---
```

Comment (saboorElahi) [reply src/pages/cplusplus/variable.md]: The header has been added!

**Comment by erictleung** [comment]
@saboorElahi LGTM! :+1: Thanks for contributing! :sparkles:

**Comment by saboorElahi** [comment]
My pleasure :)

**Pull request merged**